kufar/books/main.go

```diff
@@ -70,12 +76,23 @@ type CredentialsRequest struct {
 	Password string `json:"password"`
 }
 
+//Cache ...
+type Cache struct {
+	// defaultExpiration time.Duration
+
+	mu    sync.RWMutex
+	Items map[int]Book
+}
+
+type Cacher interface {
```

Comment:
Also, I know that it's a go convention to add `er` to interfaces, but you also could do something like:
```
type Cache interface {
```

```
type cache struct {
```
It will be more readable :)